Automattic/wp-calypso
# Allow checklist tasks to be marked as complete.

Although the server should be able to detect when a checklist task has been completed, a user might want to mark a task as done without actually completing it.

This change adds a call to update the status of the selected checklist item.

# Testing

* create a new site
* visit `/checklist/<site>`
* click the circle to the left of one of the incomplete tasks
* verify that a network POST request is sent to `/rest/v1/sites/<site_id>/checklist`
* the checklist should be rendered again with that task marked as complete
